HRC_TXING_CHECK/HRC_Txing_Check.py

```diff
@@ -30,58 +50,304 @@
 
-The 1152 seconds is the time it takes for txings to have taken enough samples (6) to
+The 1152 seconds is the worst case time it takes for txings to have taken enough samples (6) to
 determine that a storm is bad enough to trigger a shutdown.
 
-These are the checks and errors performed by this program:
+These are the checks, errors and warnings performed by this program:
 
-1)  Event Histogram SI mode loaded and it was previously loaded before without
-completing an HRC observation.
+1)  ACIS/HRC SI mode loaded and it was previously loaded  without completing an HRC observation.
 
 2) COACTS1=134
 
-        ERROR - 134 activation but no event histogram loaded
-        ERROR - 134 activation but event histogram not running
+        ERROR - 134 activation but no ACIS/HRC SI mode loaded
+        ERROR - 134 activation but  ACIS/HRC SI mode not running
 
-3)  COACTS1=134, EH loaded, EH running
+3)  COACTS1=134, ACIS/HRC SI mode  loaded and running
 
-     OK - Event Histogram is running long enough before  COACTS1=134
-     ERROR - Event Histogram not running long enough before  COACTS1=134
+     OK - ACIS/HRC SI mode  is running long enough before  COACTS1=134
+     ERROR -  ACIS/HRC SI mode not running long enough before  COACTS1=134
 
 
 4) 215PCAOF Command
-          OK - 215PCAOF and Event Histogram running
+          OK - 215PCAOF and  ACIS/HRC SI mode running
           ERROR - 215PCAOF but there wasn't any COACTS1=134
-	  ERROR - 215PCAOF EH loaded but not running
+	  ERROR - 215PCAOF ACIS/HRC SI mode  loaded but not running
                       - Allows a 1 second differential IF Stop Science occurs first.
 
+5) WARINING - The sequence of the commands to load the ACIS/HRC SI mode is missing a command
+
+6) WARNING - The timing of the commands to load the ACIS/HRC SI mode has an extra command inserted
+
+7) WARNING - Informs the user that there were no  ACIS/HRC SI mode/SCS-134 activation/215PCAOF groups
+                      in the load.
+
+Version 2.0:
+
+    H2C_0002(B) is now recognized as a valid SI mode to be used
+    when HRC is observing. The changes are backward compatible so that the other 
+    SI modes will still be recognized and processed correctly even though they might
+    no longer be used in the future. This allows regression testing using old tests and
+    a return to using the old SI modes in the future if desired.
+
+    The ACIS/HRC SI modes are no longer limited to Event Histogram type SI modes. 
+    The code comments and informational statements were changed to reflect this.
+    New SI modes can be added by creating a .dat  file with the commands and delays.
+    and editing the PB_to_Mode_Map.dat file.  The program no longer needs to be modified
+    when a new ACIS/HRC SI mode is created. Since we usually use the same SI mode
+    for HRC observations in a load, the code will check to see if the command sequence
+    for that load was already loaded and if so, will not repeat the loading.
```

Comment:
No.  This comment is speaking to the fact that when it runs across an HRC observation and identifies the ACIS SI mode used, it needs its own list of the commands for that SI mode in order to check the commanding in the load. It gets that list of commands by reading a file. If it has already read that file because that SI mode was used during a previous HRC observation, It knows it doesn't need to load it again.